core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -132,19 +134,33 @@ private TabletsMetadata buildExtents(AccumuloClient client) {
         var ranges = extents.stream().map(KeyExtent::toMetaRange).collect(toList());
         scanner.setRanges(ranges);
 
+        boolean extentsAreNull = extentsToFetch == null;
+
+        if (!extentsAreNull)
```

Comment:
If fetched columns is currently empty, then that means to fetch all columns so we do not want to add it because that would limit to fetching the single column.  Think the following will make this change.

```suggestion
        if (!fetchedColumns.isEmpty() && !extentsAreNull)
```